tests/images/azl-installer/mos/mos.yaml

```diff
@@ -0,0 +1,81 @@
+storage:
```

Comment:
is this PR just about the changes under tests/images/azl-installer?  seems like the other changes in this PR (tools/installer and Makefile) are in the liveinstaller PR.